modules/virt-creating-vm-snapshot-cli.adoc

```diff
@@ -64,7 +64,7 @@ $ kubectl wait my-vm my-vmsnapshot --for condition=Ready
 +
 [NOTE]
 ====
-Online snapshots have a default time deadline of five (`5`) minutes. If the snapshot does not complete successfully in five minutes, the status is set to `failed`. Afterwards, the file system will be thawed and the VM unfrozen but the status remains `failed` until you delete the failed snapshot image.
+Online snapshots have a default time deadline of three minutes (`3m`). If the snapshot does not complete successfully in three minutes, the status is set to `failed`. Afterwards, the file system will be thawed and the VM unfrozen but the status remains `failed` until you delete the failed snapshot image.
 
 To change the default time deadline, add the `FailureDeadline` attribute to the VM snapshot spec with the time (in minutes) that you want to specify before the snapshot operation times out.
```

Comment:
here need to change from (in minutes) to change that it is in the format of adding 's' or 'm' for seconds or minutes. without it it will be in seconds as default.

Reply:
See above comments in which I have addressed this:

_To change the default time deadline, add the `FailureDeadline` attribute to the VM snapshot spec with the time designated in minutes (`m`) that you want to specify before the snapshot operation times out._